revproxy/views.py

```diff
@@ -20,9 +20,16 @@
 
 
 class ProxyView(View):
+    #: Variable to add remote_user to request
     add_remote_user = False
+    #: Variable that represents the name of file containing
+    #: the diazo theme template
     diazo_theme_template = 'diazo.html'
+    #: Variable verify if html5 will be used
```

Comment:
This determines if the doctype of the document will be replaced by HTML5 or not.